wfl/App_Start/OwinConfiguration.cs

```diff
@@ -12,9 +15,15 @@ public class OwinConfiguration
     {
         public void Configuration(IAppBuilder app)
         {
-            HttpConfiguration configuration = new HttpConfiguration();
+            IUnityContainer container = new UnityContainer();
+            DIContainerConfig.RegisterElements(container);
+            container.RegisterType(typeof(RestaurantsController));
```

Comment:
Love this in here now.  In the next PR, I think we should try to find a way to auto-register all controllers so that we won't have to remember to add every controller here everytime we create a new one.